pkg/gatewayserver/gatewayserver.go

```diff
@@ -733,11 +738,19 @@ func (gs *GatewayServer) handleLocationUpdates(conn connectionEntry) {
 			status, _, ok := conn.StatusStats()
 			locations := status.AntennaLocations
 			antennas := conn.Gateway().Antennas
```

Comment:
I don't think we really need `locations` and `antennas` anymore

Reply:
Removed them, using `status.AntennaLocations` and `conn.Gateway().Antennas` directly.